<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-LSIEJKN" ignored="0" name="t2" tests="5" failures="2" timestamp="2020-11-11T08:55:04 CST" time="0.034" errors="0">
  <testcase name="test1" time="0.0" classname="com.webtest.demo.FreeMarker"/>
  <testcase name="test2" time="0.0" classname="com.webtest.demo.FreeMarker"/>
  <testcase name="test3" time="0.0" classname="com.webtest.demo.FreeMarker">
    <failure type="java.lang.AssertionError" message="expected [1] but found [13]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [13]
at com.webtest.demo.FreeMarker.test3(FreeMarker.java:27)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test4" time="0.0" classname="com.webtest.demo.FreeMarker"/>
  <testcase name="test5" time="0.0" classname="com.webtest.demo.FreeMarker">
    <failure type="java.lang.AssertionError" message="expected [11] but found [13]">
      <![CDATA[java.lang.AssertionError: expected [11] but found [13]
at com.webtest.demo.FreeMarker.test5(FreeMarker.java:39)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test5 -->
</testsuite> <!-- t2 -->
